webviews/editorWebview/index.css

```diff
@@ -1382,3 +1382,31 @@ code {
 .blob-code-marker-deletion::before {
 	content: "- ";
 }
+
+.markdown-alert {
+	padding: .5rem .5rem;
+	margin-bottom: 1rem;
+	color: inherit;
+	border-left: .25em solid var(--vscode-editorWarning-foreground);
+}
```

Comment:
Do you need to add support for other notes? https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts